chingu-coders/Voyage2-Turtles-02
# page speed feature

## Purpose
* Create page speed feature to test speed scores for desktop and mobile

## Testing
* Pull code and test in your local device
* Ensure that the page speed box opens up
* enter a url and check to see that it provides results
* check to see that results can be shown for both mobile and desktop views
* when results are shows, scroll through and click any of the 'more details' buttons to see more
* click 'less details' to minimize the extra content

## Still need to do:
* Show an error when an invalid URL is entered
